modules/ROOT/pages/create-an-api-configure-service-mesh-CRD.adoc

```diff
@@ -95,60 +95,65 @@ The API that was created is displayed.
 
 Anypoint Service Mesh supports the use of Kubernetes secrets to store and manage sensitive information, such as your user credentials.
 
+*Task Prerequisites*
+
+You must have the Devops or Admin role or permission to create the Kubernetes secret.
+//[PA: Rodrigo--Are these the exact nams of the roles or permissions?]
+
 To store your user credentials in Kubernetes secrets:
 
-. Create a CRD with the following Kubernetes secret details in a .yaml file, for example `user-secret.yaml`:
+. In a `.yaml` file, create a secret in which to store the sensitive information, for example `orders-api-config.yaml`:
 +
 [source,text,linenums]
 ----
 apiVersion: v1
 kind: Secret
 metadata:
-  name: my-credentials
-  namespace: nto-payment
+  name: <secret name> # example: orders-api-config
+  namespace: <namespace name> # example: default
 type: Opaque
 stringData:
-  user-pass:
-    '{
-      "user": "<user>",
-      "password": "<password>"
-    }'
+  config: |
+    {
+      "user": "<replace-with-anypoint-username>",
+      "password": "<replace-with-anypoint-clientSecret>"
+    }
 ----
-. Apply the CRD using the following command:
+. Apply the secret: 
 +
-`kubectl apply -f user-secret.yaml`
+`kubectl apply -f orders-api-config.yaml`
 +
-If the secret is correctly applied, a “secret/my-credentials configured” message is displayed. 
+If the secret is correctly applied, a "secret/orders-api-config created" message is displayed.
 +
-. Reference the secret in your API CRD, instead of the user and password parameters, as illustrated in the following example:
+. Reference the secret in your API CRD, (instead of the user and password parameter), as illustrated in the following example:
 +
 [source,text,linenums]
 ----
 apiVersion: servicecatalog.k8s.io/v1beta1
 kind: ServiceInstance
 metadata:
-  name: inventory-resource-api
-  namespace: nto-payment
-  labels: 
+  name: <api name> # example: orders-api
+  namespace: <namespace name> # example: default
+  labels:
     service-mesh.mulesoft.com/kind: api
 spec:
   clusterServiceClassExternalName: anypoint-platform-api-instance
   clusterServicePlanExternalName: service-mesh
   parameters:
-    environmentId: 28f63f07-7422-44f4-9e95-a85fa58f8e9c
-    groupId: 380c89c6-8421-4c66-85b3-80336d9cb1ca
-    assetId: inventory-api
-    version: 1.0.0
-    instanceLabel: public
+    environmentId: <environment ID> # example: 7cea45c6-3237-464d-bd34-3122c4abde29
+    groupId: <group ID> # example: 350b8164-e809-4091-91f5-079fb30e5a1e
+    assetId: <asset ID> # example: orders-api
+    version: <version> # example: 0.0.1
+    instanceLabel: <instance label> # example: public
     tags:
       apiSpec:
-      - spec1
+      - <tag in Exchange asset> # example: orders
       apiInstance:
-      - spec2
-  parametersFrom:
-  - secretKeyRef:
-      name: my-credentials
-      key: user-pass
+      - <tag in API Manager instance> # example: orders
+  parametersFrom: # we will refer to the secret holding the config from here
```

Comment:
"the secret holding the config from here"?
I know this is just an internal comment, but it's pretty unclear, at least to me....

Reply:
We need to find some way to refer to the previous created secrets

Reply:
The element specifies the secret name and configuration required for retrieving the parameters.